ibm_vpd_app.cpp

```diff
@@ -918,6 +923,26 @@ static void populateDbus(T& vpdMap, nlohmann::json& js, const string& filePath)
         const auto& objectPath = item["inventoryPath"];
         sdbusplus::message::object_path object(objectPath);
 
+        vector<string> ccinList;
+        if (item.find("ccin") != item.end())
+        {
+            for (const auto& cc : item["ccin"])
+            {
+                string ccin = cc;
+                transform(ccin.begin(), ccin.end(), ccin.begin(), ::toupper);
+                ccinList.push_back(ccin);
+            }
+        }
+
+        if (!ccinFromVpd.empty() && !ccinList.empty() &&
+            (find(ccinList.begin(), ccinList.end(), ccinFromVpd) ==
+             ccinList.end()))
+        {
+            cout << "CCIN " << ccinFromVpd << " not found, skip exposing port "
```

Comment:
this code could be used for all sorts of things, not just exposing port objects.  so you might want to remove the reference to ports.

beyond that though, this just feels like noise to me.  the code is correctly doing what it is supposed to do - do we really need to be putting that on the console?

Reply:
Will remove.

Reply:
+1 please remove the cout